api/scpca_portal/common.py

```diff
@@ -16,6 +16,8 @@
 INPUT_DATA_PATH = DATA_PATH / "input"
 OUTPUT_DATA_PATH = DATA_PATH / "output"
 
+EXPECTED_VALUES_PATH = CODE_PATH / "scpca_portal" / "test" / "expected_values"
+
```

Comment:
```suggestion
```
This is a test value and we do plan to overhaul this file and move these types of values into an environment config which would be more appropriate. So let's just remove this for now and define it in the test.